sklearn/tree/tree.py

```diff
@@ -1018,6 +1132,9 @@ class DecisionTreeRegressor(BaseDecisionTree, RegressorMixin):
         When using either a smaller dataset or a restricted depth, this may
         speed up the training.
 
+    alpha : float, optional (default=0.0)
```

Comment:
This is not a sufficiently descriptive parameter name.

Reply:
Agreed. It would need a prefix to describe it, maybe `cost_complexity_alpha`?